cves/CVE-2014-0098.yml

```diff
@@ -142,10 +149,15 @@ interesting_commits:
     interesting in light of the lessons learned from this vulnerability. Any
     emerging themes?
   commits:
-    - commit:
-      note:
-    - commit:
-      note:
+    - commit: 577eaf25cdc37727001d83a858b75c11687ad745
+      note: |
+        This commit reverts a previous commit that affects the method containing
+        the vulnerability to get versions 2.2/2.4/2.x in sync for a "single fix"
+        for the CVE, as that was the following commit.
+    - commit: 82756fe342d55c97d3c2de33503612fb0017d59d
+      note: |
+        Commit reverted by commit above, ensures that all cookies have a name
```

Comment:
"commit reverted by commit above" reads weirdly. Maybe refactoring it would fix it?